hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/RBFMetrics.java

```diff
@@ -715,8 +716,12 @@ public String getTopTokenRealOwners() {
     RouterSecurityManager mgr =
         this.router.getRpcServer().getRouterSecurityManager();
     if (mgr != null && mgr.getSecretManager() != null) {
-      return JSON.toString(mgr.getSecretManager()
-          .getTopTokenRealOwners(this.topTokenRealOwners));
+      try {
+        return JsonUtil.toJsonString((mgr.getSecretManager()
```

Comment:
This would read nicer if extracted.

Reply:
@goiri I have pushed the changes successfully from local for the recent comments, but somehow those are not appearing on this PR